### ISSUE TITLE
Social Sharing

### ISSUE THREAD

**Issue by juniortimeline** [PHPfox-Official/phpfox-v4-feature-requests#313]
We need to return back the social sharing module. PhpFox is Social Network software and this module is very important to be more social. Thank You

**Comment by chena321** [comment]
yes.  bring this back so it come close to v3

**Comment by chena321** [comment]
Now I'm using addThis on events, blogs section to share on fb, twitter, whatsapp but its get blocked by adBlocker.  Take adBlockers in consideration. 

Also should be able to use the sharing on custom modules/pages.

**Comment by neil96** [comment]
Hi @chena321 ,

We have marked this feature for version 4.6.0. Some recently released apps have this feature, but many are still unsupported. Therefore, the supporting documentation is not sufficient. If you have any difficulties in using, please [send us a ticket](https://clients.phpfox.com/submitticket.php). If you think this is a bug, please post a new issue [here](https://github.com/PHPfox-Official/phpfox-v4-issues/issues/new).

Regards.

**Comment by juniortimeline** [comment]
Thanks @neil96

**Comment by neil96** [comment]
Hi @juniortimeline ,

We support this feature via addthis feature.

Regards.
